components/Layout.tsx

```diff
@@ -1,31 +1,30 @@
 import { ReactNode } from 'react'
 import Head from 'next/head'
 import Sidebar from './Sidebar'
-import { useTheme } from 'contexts/theme'
 
 const Layout = ({ children }: { children: ReactNode }) => {
-  const theme = useTheme()
-
   return (
-    <div
-      className={`h-screen ${theme.isDarkTheme ? 'bg-dark' : 'bg-white'} ${
-        theme.isDarkTheme ? 'text-gray/75' : 'text-dark-gray/75'
-      }`}
-    >
+    <div className="h-screen relative overflow-hidden juno-gradient-bg text-white">
       <Head>
         <title>JunoTools</title>
         <meta name="description" content="Tooling dApp for Juno Network" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <main className="h-full">
-        <div className="h-full flex">
-          <Sidebar />
-          <div className="flex justify-center items-center h-full w-full">
-            {children}
-          </div>
-        </div>
-      </main>
+      {/* eslint-disable-next-line @next/next/no-img-element */}
+      <img
+        src="/confetti.svg"
+        alt="plumbus confetti"
+        className="absolute pointer-events-none h-[calc(100vh+180px)] top-0 right-0"
+      />
+
+      <div className="h-full flex z-10">
+        <Sidebar />
+        <main className="flex-grow flex flex-col items-center justify-center">
+          {children}
+          {/*  */}
```

Comment:
What is this line doing?

Reply:
Mostly for placeholder so when adding/removing lines, git will show the diff replacing the placeholder. But we can remove this if you'd like:

```suggestion
```